Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
> What file is ```python --version``` getting the number from originally

It's from PY_VERSION macro of patchlevel.h. In ```install``` step, I can just read ```Include/patchlevel.h```. However I don't know how to access the source tree in ```post_install```. Is it possible? In installed files, patchlevel.h can be found in following paths:
```
$ find -L /usr/local -name patchlevel.h                                                                                    
/usr/local/Cellar/python3/HEAD-da510d1aa683_3/Frameworks/Python.framework/Versions/3.7/Headers/patchlevel.h
/usr/local/Cellar/python3/HEAD-da510d1aa683_3/Frameworks/Python.framework/Versions/3.7/include/python3.7m/patchlevel.h
/usr/local/Frameworks/Python.framework/Versions/3.7/Headers/patchlevel.h
/usr/local/Frameworks/Python.framework/Versions/3.7/include/python3.7m/patchlevel.h
/usr/local/opt/python3/Frameworks/Python.framework/Versions/3.7/Headers/patchlevel.h
/usr/local/opt/python3/Frameworks/Python.framework/Versions/3.7/include/python3.7m/patchlevel.h
/usr/local/var/homebrew/linked/python3/Frameworks/Python.framework/Versions/3.7/Headers/patchlevel.h
/usr/local/var/homebrew/linked/python3/Frameworks/Python.framework/Versions/3.7/include/python3.7m/patchlevel.h
```
All requires knowing "3.7" first.